tgui/packages/tgui/interfaces/TachyonArray.js

```diff
@@ -0,0 +1,110 @@
+import { useBackend } from '../backend';
+import { Button, Flex, Table, NoticeBox, Section, LabeledList } from '../components';
+import { Window } from '../layouts';
+
+export const TachyonArray = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    records = [],
+    explosion_target,
+    toxins_tech,
+    printing,
+  } = data;
+  return (
+    <Window>
+      <Window.Content scrollable>
+        <Section>
+          <LabeledList>
+            <LabeledList.Item label="Shift's Target">
+              {explosion_target}
+            </LabeledList.Item>
+            <LabeledList.Item label="Current Toxins Level">
+              {toxins_tech}
+            </LabeledList.Item>
+            <LabeledList.Item label="Administration">
+              <Button
+                icon="print"
+                content="Print All Logs"
+                disabled={!records.length || printing}
+                align="center"
+                onClick={() => act('print_logs')} />
+              <Button.Confirm
+                icon="trash"
+                content="Delete All Logs"
+                disabled={!records.length}
+                color="bad"
+                align="center"
+                onClick={() => act('delete_logs')} />
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        {!records.length ? (
+          <NoticeBox>
+            No Records
+          </NoticeBox>
+        ) : (
+          <TachyonArrayContent />
+        )}
+      </Window.Content>
+    </Window>
+  );
+};
+
+export const TachyonArrayContent = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    records = [],
+  } = data;
+
+  return (
+    <Section>
+      <Flex>
+        <Flex.Item>
+          <Section label="Logged Explosions">
```

Comment:
The correct syntax is `title="Logged Explosions"`. But also, I think this needs to go onto the outer section at line 60. Otherwise, each time a test is made there will be a section named "Logged Explosions" created. 

And for a personal opinion, I think you should ditch this section, because nested sections always looks a bit funky with their double borders. 

![image](https://user-images.githubusercontent.com/42044220/96773049-9e3b0180-13a9-11eb-80c5-34686262bb74.png)

Reply:
Oh darn forgot to change that one. Good spot will fix